src/sentry/static/sentry/app/routes.jsx

```diff
@@ -860,6 +861,16 @@ function routes() {
           component={errorHandler(LazyLoad)}
         />
       </Route>
+      {/* A route tree for lightweight organizational detail views */}
+      <Route path="/:orgId/" component={errorHandler(LightWeightOrganizationDetails)}>
```

Comment:
This route jused to be after the `routes:organization-root` hook. Does the getsentry hook for that key still run now?

Reply:
I believe that the hook just adds additional routes so this should be okay